shanoir-ng-datasets/src/main/java/org/shanoir/ng/configuration/ShanoirDatasetsConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package org.shanoir.ng.configuration;
+
+import java.util.concurrent.Executor;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.EnableAsync;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+@Configuration
+@EnableAsync
+public class ShanoirDatasetsConfiguration {
```

Comment:
What is it used for ?

Reply:
Hi Jean-Come, I removed it. I think it was an experiment, when I thought about Spring Batch...